src/components/Iwd/ActivityFeed.vue

```diff
@@ -0,0 +1,51 @@
+<template>
+	<div class="tw-flex tw-gap-x-0.5 tw-overflow-x-auto tw-py-2 tw-px-1 hide-scrollbar">
+		<activity-card
+			v-for="(activity, index) in activities"
+			:key="index"
+			:activity="activity"
+		/>
+	</div>
+</template>
+
+<script>
+import ActivityCard from '@/components/Iwd/ActivityCard';
+import IwdActionsQuery from '@/graphql/query/IwdActions.graphql';
+
+export default {
+	name: 'ActivityFeed',
+	components: {
+		ActivityCard,
+	},
+	inject: ['apollo'],
+	data() {
+		return {
+			activities: [],
+		};
+	},
+	methods: {
+		fetchActivities() {
+			// Fetch activities from API
+			this.apollo.query({
```

Comment:
Do you think we need a loader?

Reply:
I considered this but should be good to have dynamic data to test this to see how is working. Locally is looking fine, but a good call to test in dev.
An edge case is returning non-data, so, no sure what is better, showing a loader and then removing this or not showing a loader and then show the feed if having data...